models/timeswap/polygon/timeswap_polygon_borrow.sql

```diff
@@ -0,0 +1,199 @@
+{{ config(
+    alias = 'borrow',
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['Transaction_Hash'],
```

Comment:
We may need to rename all columns to strictly follow [lower_snake_case](https://en.wikipedia.org/wiki/Snake_case).